src/command/defaults/TempBanCommand.php

```diff
@@ -0,0 +1,56 @@
+<?php
+
+declare(strict_types=1);
+
+namespace pocketmine\command\defaults;
+
+use pocketmine\command\Command;
+use pocketmine\command\CommandSender;
+use pocketmine\command\utils\InvalidCommandSyntaxException;
+use pocketmine\lang\KnownTranslationFactory;
+use pocketmine\lang\KnownTranslationKeys;
+use pocketmine\permission\BanEntry;
+use pocketmine\permission\DefaultPermissionNames;
+use pocketmine\player\Player;
+use pocketmine\utils\AssumptionFailedError;
+
+class TempBanCommand extends Command{
+	public function __construct(string $name){
+		parent::__construct(
+			$name,
+			KnownTranslationKeys::POCKETMINE_COMMAND_TEMP_BAN_PLAYER_DESCRIPTION,
+			KnownTranslationKeys::COMMANDS_TEMP_BAN_USAGE
```

Comment:
These don't seem to exist.
Also, vanilla doesn't have a /tempban command, so this translation should have a `pocketmine.` prefix.